WordPress/src/main/java/org/wordpress/android/ui/comments/CommentActions.java

```diff
@@ -437,7 +437,8 @@ public void run() {
      */
     private static void deleteComment(final int accountId,
                                         final Comment comment,
-                                        final CommentActionListener actionListener) {
+                                        final CommentActionListener actionListener,
+                                        final boolean forceDelete) {
```

Comment:
This feels nitpicky, but I think the `forceDelete` parameter should be named `deletePermanently` for clarity.